samples/kotlin-android-app/build.gradle

```diff
@@ -24,11 +26,12 @@ android {
 
     buildTypes {
         release {
-            minifyEnabled false
+            minifyEnabled true
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
         }
     }
     compileOptions {
+        coreLibraryDesugaringEnabled true
```

Comment:
Why do we need to enable this? I have updated my local to Gradle v8.0 and it somehow works without it